src/app/ipv/middleware.js

```diff
@@ -213,7 +222,6 @@ function isValidPage(pageId) {
 }
 
 function handleAppStoreRedirect(req, res, next) {
-  // PYIC-4816 - consider whether we should override this with the current request's sniffed device type.
```

Comment:
For the record, this now has it's own ticket PYIC-5560 so this comment isn't needed